client/src/components/course.jsx

```diff
@@ -1,40 +1,20 @@
 import React from "react";
+import ReactStars from 'react-stars';
 import Step from './step.jsx';
 
-class Course extends React.Component {
-
-  constructor(props) {
-    super(props);
-    this.state = {
-    };
-  }
-
-
-  
-  render(){
-
-  const courseId = this.props.match.params.id;
-
+const Course = (props) => {
+  const courseId = props.match.params.id;
   const course = sampleData[courseId];
+  const { steps, name, rating, description } = course;
 
-  const steps = course.steps;
-
-  const lessonPlan = steps.map((step) => {
-      return (
-        <Step
-          key={step.id}
-          data={step}/>
-      )
-    });
-    return (
-      <div className="course-view">
-          <h3>{course.name}</h3>
-          <h4>Rating: {course.rating}</h4>
-          <p>{course.description}</p>
-          {lessonPlan}
-      </div>
-    );
-  }
+  return (
+    <div className="course-view">
+      <h3>{name}</h3>
+      <ReactStars value={rating} size={20} edit={false}/>
```

Comment:
Clean up looks good. Didn't we decide to implement ratings as a subcomponent? Take a look at the courseHeader.jsx and enroll.jsx in my pull request. Idea is that it would go in the courseHeader as well. (I'm open to doing this differently, but I'm pretty sure that's what we discussed?)

Also I'm hesitant to merge this cleanup to course.jsx until Jeff is done.